tests/regression_tests/plot/test.py

```diff
@@ -18,6 +18,15 @@ def __init__(self, plot_names):
     def _run_openmc(self):
         openmc.plot_geometry(openmc_exec=config['exe'])
 
+        # TEMP: this should always be checked once vtk is added
+        # to Python3.7
+        try:
+            import vtk
+            call(['../../../scripts/openmc-voxel-to-vtk'] +
+                 glob.glob('plot_4.h5'))
+        except:
+            pass
```

Comment:
Actually `subprocess.run` wasn't added until 3.5, so this would break the script for 3.4. However, you can use [check_call](https://docs.python.org/3/library/subprocess.html#subprocess.check_call) which is equivalent to `run(..., check=True)`.

Reply:
And let's make the except clause more specific: `except ImportError` so that actual errors don't pass silently.

Reply:
I'll move to `check_call` in the next commit. Thanks!

Good point, will become a proper skip check now that it's going to be its own test though.